modules/ROOT/pages/status-updates.adoc

```diff
@@ -4,15 +4,19 @@ When working with APIs and the unified schema, status updates let you know when
 
 == Status: Updating
 
-You can add to and update the unified schema at any time and apply changes. After you apply changes, the unified schema will enter an “updating” state, and the status will indicate *Updating*. This status update is visible to anyone in the business group with permissions for the environment.
+You can add to and update the unified schema at any time and apply changes. After you apply changes, the unified schema enters an “updating” state, and the status indicates the progress and queued updates. This status update is visible to anyone in the business group with permissions for the environment.
```

Comment:
Do we need the first "and apply changes"? Does "add to and update" not imply that? Your choice, of course:
```suggestion
You can add to and update the unified schema at any time. After you apply changes, the unified schema enters an `updating` state during which you can view the status of in-progress and queued updates. This status update is visible to anyone in your business group who has permissions for the environment.
```